pkg/contract/contract.go

```diff
@@ -630,3 +630,24 @@ func UnpackLog(
 	contract := bind.NewBoundContract(common.Address{}, *abi, nil, nil, nil)
 	return contract.UnpackLog(event, eventName, log)
 }
+
+// GetContractOwner gets owner for https://docs.openzeppelin.com/contracts/2.x/api/ownership#Ownable-owner contracts
+func GetContractOwner(
```

Comment:
What is this used for? And what determines which contract addresses we call this on?

Reply:
This is to see who can make changes to a PoA L1 since only the holder of the private key to validator manager controller address can make changes like add and remove validator. Since validator manager contracts are openzeppelin contracts we can use https://docs.openzeppelin.com/contracts/2.x/api/ownership#Ownable-owner to see which address is controlling the validator manager contract.